api/src/test/resources/mock-data.xml

```diff
@@ -632,13 +632,13 @@
     <keyword><id>196</id><creationDate>2021-06-14 14:44:20.602561</creationDate><updateDate>2021-06-14 14:44:20.602561</updateDate><keywordTypeEnum>Alignment</keywordTypeEnum><value>Accounting Systems</value><uri>2021-06-14 14:44:20.602561</uri></keyword>
   </keywords>
   <collections>
-    <collection><id>1</id><uuid>975b5c26-9506-4e73-b4b2-83f4a838529d</uuid><name>Power_Skills_Framework</name><status>Published</status><publishDate>2021-10-01 09:00:00.000000</publishDate><skillsIds>50,64,53,63,55,61,57</skillsIds><skillsCount>7</skillsCount><author>8</author></collection>
-    <collection><id>2</id><uuid>9124e022-0723-4bf3-8bec-0a271616b5ea</uuid><name>SEL</name><status>Published</status><publishDate>2021-10-01 09:00:00.000000</publishDate><skillsIds>50,64,63,61,57,55,53</skillsIds><skillsCount>7</skillsCount><author>8</author></collection>
-    <collection><id>3</id><uuid>afaa6b8a-680a-4642-add1-014f977c9ca7</uuid><name>21st_Century_Skills</name><status>Published</status><publishDate>2021-10-01 09:00:00.000000</publishDate><skillsIds>55,57,61,53,63,64,50</skillsIds><skillsCount>7</skillsCount><author>8</author></collection>
-    <collection><id>4</id><uuid>9903442f-ed1e-42d4-8d95-858a26204d50</uuid><name>0</name><status>Published</status><publishDate>2021-10-01 09:00:00.000000</publishDate><skillsIds>50,167,168,55,61,53,57</skillsIds><skillsCount>7</skillsCount><author>8</author></collection>
-    <collection><id>5</id><uuid>f07b2276-4d1c-46cf-a971-746c214bc3bc</uuid><name>Health_Open_Skills</name><status>Published</status><publishDate>2021-10-01 09:00:00.000000</publishDate><skillsIds>168,167,166,142,141,140,139,66</skillsIds><skillsCount>8</skillsCount><author>8</author></collection>
-    <collection><id>6</id><uuid>3dc22c04-1aa2-4dea-9dfc-0f016573428a</uuid><name>DEI_Collection</name><status>Published</status><publishDate>2021-10-01 09:00:00.000000</publishDate><skillsIds>78,77,75</skillsIds><skillsCount>3</skillsCount><author>8</author></collection>
-    <collection><id>7</id><uuid>2938a728-f6e2-41a1-8a3b-9a838a56d212</uuid><name>Character</name><status>Published</status><publishDate>2021-10-01 09:00:00.000000</publishDate><skillsIds>78,77,76,75</skillsIds><skillsCount>4</skillsCount><author>8</author></collection>
+    <collection><id>1</id><uuid>975b5c26-9506-4e73-b4b2-83f4a838529d</uuid><name>Power_Skills_Framework</name><status>Published</status><publishDate>2021-10-01 09:00:00.000000</publishDate><skillsIds>50,64,53,63,55,61,57</skillsIds><skillsCount>7</skillsCount><author>8</author><workspaceOwner>user@email.com</workspaceOwner></collection>
```

Comment:
It might be best to created a single new collection XML record here, as the owner name should be unique (that's not enforced as a business rule in the DB), and this workspace will not have a publish date.

Reply:
I totally agree